cf/s3-v2.yaml

```diff
@@ -0,0 +1,265 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: Set up S3 buckets for Cellenics [managed by github.com/hms-dbmi-cellenics/iac]
+
+Parameters:
+  Environment:
+    Type: String
+    Default: development
+    AllowedValues:
+      - development
+      - staging
+      - production
+    Description: The environment for which the buckets need to be created.
+
+Resources:
+  PlotsTablesBucket:
+    Type: AWS::S3::Bucket
+    Properties:
+      BucketName: !Sub "plots-tables-${Environment}-${AWS::AccountId}"
+      PublicAccessBlockConfiguration:
+        BlockPublicAcls: True
+        BlockPublicPolicy: True
+        IgnorePublicAcls: True
+        RestrictPublicBuckets: True
+      BucketEncryption:
+        ServerSideEncryptionConfiguration:
+          - ServerSideEncryptionByDefault:
+              SSEAlgorithm: 'aws:kms'
+
+  CellSetsBucket:
+    Type: AWS::S3::Bucket
+    Properties:
+      BucketName: !Sub "cell-sets-${Environment}-${AWS::AccountId}"
+      PublicAccessBlockConfiguration:
+        BlockPublicAcls: True
+        BlockPublicPolicy: True
+        IgnorePublicAcls: True
+        RestrictPublicBuckets: True
+      BucketEncryption:
+        ServerSideEncryptionConfiguration:
+          - ServerSideEncryptionByDefault:
+              SSEAlgorithm: 'aws:kms'
+
+  WorkerResultsBucket:
```

Comment:
I think these logical Ids need to be changed  to be different from the other file (s3.yaml) right? If yes how do you think I should change the name? Maybe WorkerResultsBucket-v2? or ...-global?

Reply:
why do they need changing? Is cloud formaion complaining?

Reply:
I thought they need to be unique? Nothing is complaining so far.

Reply:
Maybe they are Okay like that as long the names are unique within a template not globally

Reply:
yes, the names should be ok like that in this case. Let's keep them as they are.